app/javascript/controllers/auto_citation_controller.js

```diff
@@ -73,33 +73,17 @@ export default class extends Controller {
   }
 
   get year() {
+    if (this.hasEmbargoYearTarget && this.yearTarget.value.length < 1) {
+      return this.embargoYearTarget.value
+    }
     return this.yearTarget.value
   }
 
-  get month() {
-    return this.monthTarget.value
-  }
-
-  get day() {
-    return this.dayTarget.value
-  }
-
   get date() {
     const date = new Date();
```

Comment:
Do we still need the `Date` code since we're only dealing in years now?

Reply:
I think so since we want to return the current year if there is not an `embargoYear` or `publication_year`.